packages/teams-js/src/internal/utils.ts

```diff
@@ -338,3 +339,24 @@ export function createTeamsAppLink(params: pages.NavigateToAppParams): string {
   }
   return url.toString();
 }
+
+/**
+ * @hidden
+ * Checks if the Adaptive Card schema version is supported by the host.
+ * @param hostAdaptiveCardSchemaVersion Hosts supported Adaptive Card Version in the runtime.
```

Comment:
The param description is a little unclear to me. Would this be an accurate description:

Adaptive Card schema version to check for host support